### ISSUE TITLE
Error: regsids is not defined

### ISSUE THREAD

**Issue by rnbplt** [MarkBaggett/srum-dump#22]
Hi,

When runnnig srum_dump2.exe on a Windows 10 machine I get the following error:

```
File "srum_dump2.py", line 563, in <module>
  if regsids:
NameError: name 'regsids' is not defined.
```

Any idea of what's going wrong?

**Comment by MarkBaggett** [comment]
Can you tell me what command line arguments you are passing to the file?

**Comment by crumbtifact** [comment]
Hi Mark, I am receiving the same error. I am using Windows 7 64-bit, and I have installed Python 3.7.4 as well.

The arguments are the following:
`srum_dump2.exe -i "C:\...." -o "C:\..." -t "C:\.."`

The errors are the following:
```
Traceback (most recent call last):
  File "srum_dump2.py", line 571, in <module>
    if regsids:
NameError: name 'regsids' is not defined
[13404] Failed to execute script srum_dump2
```

**Comment by gajos112** [comment]
Any one have solution for that ?

**Comment by MarkBaggett** [comment]
There was an error in the code that caused the program to crash if you didn't provide a copy of the SOFTWARE registry hive.   This has been resolved and it will no longer crash.   Please be aware that without the software registry hive SRUM-DUMP will not be able to resolve names of (non-standard) users and network profile names.  Features like wireless access points name resolution will not be available.   The problem is resolved in version 2.1.
